readme.md

```diff
@@ -21,22 +21,22 @@ The repositories below are all used to maintain [digital.gov:](https://digital.g
+| [18F/**dns**](https://github.com/18F/dns)                       | DNS configuration for digital.gov domains managed by GSA TTS.                                                                                            |
+| [uswds/**uswds**](https://github.com/uswds/uswds)               | This site is developed using the U.S. Web Design System v2, managed by GSA TTS.                                                                          |
 
 </details>
 
 <details>
   <summary>Tools - projects we have created in order to better aid our work on the digital.gov platform.</summary>
 
-| Project                                                                                       | Description                                                                                                                                                                                                                                                                                      |
-| --------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [GSA/**digital.gov-design**](https://github.com/GSA/digital.gov-design)      | A collection of design assets used for the digital.gov platform.                                                          |
-| [GSA/**digitalgov-workflow**](https://github.com/GSA/digitalgov-workflow)                           | A tool for managing the digital.gov editorial workflow on GitHub (**archived**).                                                                                                                                                                            |
-| [GSA/**redir**](https://github.com/GSA/redir)                   | A basic [Jekyll](https://jekyllrb.com/) template to use for temporary redirects.
+| Project                                                                                    | Description                                                                      |
+| ------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------- |
+| [GSA/**digital.gov-design**](https://github.com/GSA/digital.gov-design)                    | A collection of design assets used for the digital.gov platform.                 |
+| ~[GSA/**digitalgov-workflow**](https://github.com/GSA/digitalgov-workflow)~ (**archived**) | A tool for managing the digital.gov editorial workflow on GitHub.                |
+| ~[GSA/**redir**](https://github.com/GSA/redir)~ (**archived**)                             | A basic [Jekyll](https://jekyllrb.com/) template to use for temporary redirects. |
```

Comment:
Updated these sections for clarity. 

**Screenshot of before/after:**
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/dbee19a4-6390-45ce-947d-3cddf7d943f1)